server/basicServices/basicServices/GenericRepresentation.js

```diff
@@ -145,6 +145,8 @@ exports.getResponseValueList = function (operationName) {
                                 responseInstanceValue = value[onfAttributes.RESPONSE_PROFILE.STATIC_VALUE];
                             }
                             responseInstancedataTypeOfValue = typeof responseInstanceValue;
+                            responseInstanceValue = dataUpdatePeriod.getDataUpdatePeriodEnum(fieldName, responseInstanceValue)
```

Comment:
can we declare this method withing the file instead of creating separate file for this only function